blocks/init/src/Blocks/components/search-bar/search-bar-style.scss

```diff
@@ -5,12 +5,19 @@
 	display: flex;
 	align-items: center;
 
+	&__label {
+		margin-right: 15px;
+
+		&--hidden {
+			  @extend %visually-hidden;
+		}
+	}
+
 	&__input {
```

Comment:
This should be in a map, not directly applied.

Reply:
Extending a placeholder? 👀

Reply:
map is not mandatory to use so this is ok